addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -347,27 +353,28 @@ function factory(dependencies) {
             this._disconnectAudioMonitor && this._disconnectAudioMonitor();
             if (this.messaging.userSetting.usePushToTalk || !this.channel || !this.audioTrack) {
                 this.currentRtcSession.update({ isTalking: false });
-                return;
-            }
-            try {
-                this._disconnectAudioMonitor = await monitorAudio(this.audioTrack, {
-                    onThreshold: async (isAboveThreshold) => {
-                        this._setSoundBroadcast(isAboveThreshold);
-                    },
-                    volumeThreshold: this.messaging.userSetting.voiceActivationThreshold,
-                });
-            } catch (e) {
-                /**
-                 * The browser is probably missing audioContext,
-                 * in that case, voice activation is not enabled
-                 * and the microphone is always 'on'.
-                 */
-                this.env.services.notification.notify({
-                    message: this.env._t("Your browser does not support voice activation"),
-                    type: 'warning',
-                });
-                this.currentRtcSession.update({ isTalking: true });
+            } else {
+                try {
+                    this._disconnectAudioMonitor = await monitorAudio(this.audioTrack, {
+                        onThreshold: async (isAboveThreshold) => {
+                            this._setSoundBroadcast(isAboveThreshold);
+                        },
+                        volumeThreshold: this.messaging.userSetting.voiceActivationThreshold,
+                    });
+                } catch (e) {
+                    /**
+                     * The browser is probably missing audioContext,
+                     * in that case, voice activation is not enabled
+                     * and the microphone is always 'on'.
+                     */
+                    this.env.services.notification.notify({
+                        message: this.env._t("Your browser does not support voice activation"),
+                        type: 'warning',
+                    });
+                    this.currentRtcSession.update({ isTalking: true });
+                }
             }
+            this._updateLocalAudioTrackEnabledState();
```

Comment:
maybe duplicate call this method inside the "if" instead of adding else and changing indent of everything